src/main/java/doit/jpastudy2/repository/ArmyRepository.java

```diff
@@ -0,0 +1,14 @@
+package doit.jpastudy2.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
```

Comment:
여기 부분 이렇게 작성하셔도 동작은 하는데, 보통은 <Class명, 그 클래스의 PK 타입>으로 작성하는 편입니다.
jpa단에서 자동으로 int <-> Long 변환을 수행하지만, 처음부터 올바르게 적어주시는게 좋을 것 같습니다~
```suggestion
public interface ArmyRepository extends JpaRepository<Army, int> {
```